packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -284,7 +284,8 @@ export async function preTransformationUploadCode(userInputState: UserInputState
         await vscode.commands.executeCommand('aws.amazonq.refresh') // so that button updates
         uploadId = await uploadPayload(payloadFilePath)
     } catch (error) {
-        const errorMessage = 'Failed to upload code'
+        getLogger().error(`${error}`)
+        const errorMessage = `Failed to upload code due to ${error}`
```

Comment:
Should this be `error.message`?

Reply:
I think either is fine, probably don't need the back ticks and $, { though

Reply:
Question/Suggestion: `getLogger().error(error)` or `getLogger().error(error.message)` should work

Reply:
Moved it one line down to reuse the `errorMessage` variable instead, keeps it consistent.

Reply:
It should be `error.message`, as mentioned in the code guidelines: https://github.com/aws/aws-toolkit-vscode/blob/master/docs/CODE_GUIDELINES.md#exceptions

The stacktrace is minified, thus it's just noise (in both the logs and telemetry).

Reply:
Updated it.